<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SceneSelectionWindow</class>
 <widget class="QWidget" name="SceneSelectionWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>617</width>
    <height>423</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>选择作业场景</string>
  </property>
  <widget class="QPushButton" name="top_button">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>180</y>
     <width>150</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	font: 28pt &quot;黑体&quot;;
    border: 2px solid blue;
    border-radius: 10px;
}
QPushButton:hover {
    border: 2px solid red;
    border-radius: 10px;
}

QPushButton:pressed {
    border: 2px solid green;
    border-radius: 10px;
}</string>
   </property>
   <property name="text">
    <string>顶板</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cant_button">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>180</y>
     <width>150</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	font: 28pt &quot;黑体&quot;;
    border: 2px solid blue;
    border-radius: 10px;
}
QPushButton:hover {
    border: 2px solid red;
    border-radius: 10px;
}

QPushButton:pressed {
    border: 2px solid green;
    border-radius: 10px;
}</string>
   </property>
   <property name="text">
    <string>斜板</string>
   </property>
  </widget>
  <widget class="QPushButton" name="side_button">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>180</y>
     <width>150</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	font: 28pt &quot;黑体&quot;;
    border: 2px solid blue;
    border-radius: 10px;
}
QPushButton:hover {
    border: 2px solid red;
    border-radius: 10px;
}

QPushButton:pressed {
    border: 2px solid green;
    border-radius: 10px;
}</string>
   </property>
   <property name="text">
    <string>侧板</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
